### ISSUE TITLE
BN的均值以及方差中分母为啥是（batchsize * width * height）

### ISSUE THREAD

**Issue by RokeAbbey-chen** [ChenGuangbinTHU/Tensorflow-BatchNorm#1]
你好，问你一个问题。
我看了好多的BN的资料，我所理解的BN中均值以及方差的分母应该是batchsize，但是你的代码中, tf.nn.moments(input, [0, 1, 2])第二项参数表明你的均值和方差的分母是 batchsize * width * height 请问这个不应该是固定是batchsize吗？
希望你能在闲暇之时为我解答，谢谢
